Objective-C/CBLReplicatorConfiguration.h

```diff
@@ -78,6 +78,11 @@ typedef BOOL (^CBLReplicationFilter) (CBLDocument* document, CBLDocumentFlags fl
  */
 @property (nonatomic, nullable) NSDictionary<NSString*, NSString*>* headers;
 
+/*
+ Specific network interface (e.g. en0 and pdp_ip0) for connecting to the remote target.
+ */
+@property (nonatomic, nullable) NSString* networkInterface;
```

Comment:
We could also enhance this to support both Interface Name and IP Address in the future.